docs/pages/config-plugins/mods.mdx

```diff
@@ -440,46 +344,11 @@ We support this to make testing, and plugin authoring easier, but we don't expec
   ]}
 />
 
-### Raw functions
-
-Expo config objects also support passing functions as-is to the `plugins` array. This is useful for testing, or if you want to use a plugin without creating a file.
-
-```js app.config.js
-const withCustom = (config, props) => config;
-
-const config = {
-  plugins: [
-    [
-      withCustom,
-      {
-        /* props */
-      },
-    ],
-    /* @info Without props */
-    withCustom,
-    /* @end */
-  ],
-};
-```
-
-One caveat to using functions instead of strings is that serialization will replace the function with the function's name. This keeps **manifests** (kinda like the **index.html** for your app) working as expected.
-
-Here is what the serialized config would look like:
-
-```json
-{
-  "plugins": [["withCustom", {}], "withCustom"]
-}
-```
-
 ## Why app.plugin.js for plugins
 
-Config resolution searches for a file named **app.plugin.js** first when a Node module ID is provided as a plugin.
-This is because Node environments are often different to Android, iOS, or web JS environments and therefore require different transpilation presets (for example, `module.exports` instead of `import/export`).
+Config resolution searches for a file named **app.plugin.js** first when a Node module ID is provided as a plugin. This is because Node environments are often different to Android, iOS, or web JS environments and therefore require different transpilation presets (for example, `module.exports` instead of `import/export`).
```

Comment:
```suggestion
Config resolution searches for a file named **app.plugin.js** first when a Node module name is provided as a plugin. This is because Node environments are often different compared Android, iOS, or web JS environments and therefore require different transpilation presets (for example, `module.exports` instead of `import/export`).
```